SUBMISSION_GUIDELINES.md

```diff
@@ -0,0 +1,17 @@
+# Fiveable Open Source Project Submission Guidelines
+
+- Projects must be constructive and unique, repos that are intended to serve as a personal website or documentation websites are not allowed. The exception to this are external guides for libraries that don't provide a guide. 
+- Projects must have a clear CODE_OF_CONDUCT.md that clearly denounces any discriminatory, unwelcoming, or any otherwise harmful behavior that damages/hinders the positive environment we've strived to create.
+- Projects must have a clear/organized README with information as to what the projects purpose is. It doesn't have to be a fancy readme, but it does need to cover basic information.
+	- repo description 
+	- clear instructions as to how to setup a local environment for contributors
+	- (encouraged) gifs/photos/text demonstrating example usage
+	- (encouraged) demo
+	- repo structure explanation (if isn't self-explanatory)
+- Projects must be contributable, cannot be archived or privated, and must be licensed under an appropriate license that is Open Source foundation certified. You can find an abiding license at (https://choosealicense.com/)
+- Projects must be submitted by either the repo owner or someone with the "maintainer" status on said repo. "contributor" status does not meet this criteria.
+- If tht well with our environment. Toxic or unwelcoming behavior is not tolerated at all.e project is a library/API, its routes/methods must be documented to a reasonable extent.
+- Submitter must display behavior that will fit well with our environment. Toxic or unwelcoming behavior is not tolerated at all.
+- If the project is a fork, it must be transformative to an extent (??).
```

Comment:
Maybe we could evaluate these types of projects on a case-by-case basis. Also the `(??)`